jekyll/_cci2/configuration-reference.md

```diff
@@ -674,10 +674,25 @@ Key | Required | Type | Description
 docker_layer_caching | N | boolean | set this to `true` to enable [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching/) in the Remote Docker Environment (default: `false`)
 {: class="table table-striped"}
 
-***Notes***:
+**Notes**:
+
 - A paid account is required to access Docker Layer Caching. Customers on paid plans can request access by [opening a support ticket](https://support.circleci.com/hc/en-us/requests/new). Please include a link to the project on CircleCI) with your request.
 - `setup_remote_docker` is not compatible with the `machine` executor. See [Docker Layer Caching in Machine Executor]({{ site.baseurl }}/2.0/docker-layer-caching/#machine-executor) for information on how to enable DLC with the `machine` executor.
 
+###### **`version`**
+
+If your job requires a specific docker version, you can set it with the
+`version` attribute.
```

Comment:
~ could we link off the to version list on the docker page here?

Reply:
Good idea. Done!